locale/en/docs/guides/backpressuring-in-streams.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Backpressuring in Streams
+layout: docs.hbs
+---
+
+# Backpressuring in Streams
+ 
+The purpose of this guide will describe what backpressure is and the 
+problems in Streams that it solves. The first part of this guide will cover 
+these questions. The second part of the guide will introduce suggested best 
+practices to ensure your application's code is safe and optimized.
+
+We assume a little familiarity with the general definition of 
+[`backpressure`][], [`Buffer`][] and [`EventEmitters`][] in Node.js, as well as 
+some experience with [`Streams`][]. If you haven't read through those docs, 
+it's not a bad idea to take a look at the API documentation first, as it will 
+help expand your understanding while reading this guide and for following along 
+with examples.
+
+## The Problem with Streams
+
+In a computer system, data is transferred from one process to another through 
+pipes and signals. In Node.js, we find a similar mechanism and called 
+[`Streams`][]. Streams are great! They do so much for Node.js and almost every
+part of the internal codebase utilizes the `Streams` class. As a developer, you 
+are more than encouraged to use them too!
```

Comment:
`Streams` isn't really a thing. There is no `Streams` module or `Streams` class. Make it clear what exactly you are referring to for each instance of `[Streams][]`:

- the `stream` module
- streams as a concept
- `Readable` or `Writable` classes

For this paragraph, I'd say something like this:

```markdown
In Node.js, we find a similar mechanism called streams in the [`stream`
module][]. Streams are great! They do so much for Node.js and almost every part
of the internal codebase utilizes that module.
```

Ditto below.